### ISSUE TITLE
Deprecated Aurerm terraform resources

### ISSUE THREAD

**Issue by Bermanian** [CheckPointSW/CloudGuardIaaS#384]
Hello Team,

I am deploying module "vmss-existing-vnet" the to the Azure, got this warning in Plan:

```
│ Warning: Deprecated Resource
│ 
│   with module.shared.module.cloudguard_fw.azurerm_virtual_machine_scale_set.vmss,
│   on shared/modules/cloudguard/main.tf line 221, in resource "azurerm_virtual_machine_scale_set" "vmss":
│  221: resource "azurerm_virtual_machine_scale_set" "vmss" {
│ 
│ The 'azurerm_virtual_machine_scale_set' resource has been superseded by the
│ 'azurerm_linux_virtual_machine_scale_set' and
│ 'azurerm_windows_virtual_machine_scale_set' resources. Whilst this resource
│ will continue to be available in the 2.x and 3.x releases it is
│ feature-frozen for compatibility purposes, will no longer receive any
│ updates and will be removed in a future major release of the Azure
│ Provider.
```

Is there an update planned for this issue?
